Source/Core/DolphinQt/Config/WiimoteControllersWidget.cpp

```diff
@@ -207,6 +232,18 @@ void WiimoteControllersWidget::ConnectWidgets()
   }
 }
 
+void WiimoteControllersWidget::OnBluetoothPassthroughDeviceChanged(int index)
+{
+  if (index == -1)
+    return;
```

Comment:
This check can be removed if you add the previously mentioned `Automatic` item to the list.